_data/teachers.yml

```diff
@@ -146,5 +146,5 @@
 - institution: George Fox University, United States
   name: J. Walker Orr
   location:
-    longitude: 45.303216
-    latitude: -122.968487
+    latitude: 45.303216
+    longitude: -122.968487
```

Comment:
Would you mind swapping the numbers instead, so that it remains consistent with the other entries?

Reply:
Yep, no problem.